tests/unit/reconcile/test_recon_capture.py

```diff
@@ -134,7 +134,7 @@ def test_recon_capture_start(mock_workspace_client, mock_spark):
     assert row.recon_metrics.schema_comparison is True
     assert row.run_metrics.status is False
     assert row.run_metrics.run_by_user == "remorph"
-    assert row.run_metrics.exception_message == ""
+    assert row.run_metrics.exception_message == ''
```

Comment:
we have to be consistent if we are going to use `single` use everywhere `double` use everywhere, the previous line has `double`

Reply:
done